erpnext/accounts/doctype/bank/bank_dashboard.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import unicode_literals
+
+from frappe import _
+
+
+def get_data():
+	return {
+		'fieldname': 'bank',
+		'transactions': [
+			{
+				'label': _('Bank Deatils'),
+				'items': ['Bank Account', 'Bank Statement Transaction Entry', 'Bank Guarantee']
```

Comment:
Remove 'Bank Statement Transaction Entry', it is not used anymore. I will write a patch to delete all related docs.